src/logic/operate.js

```diff
@@ -0,0 +1,23 @@
+import Big from 'big-js';
+
+const operate = (numberOne, numberTwo, operations) => {
+  const [one, two] = [Big(numberOne), Big(numberTwo)];
+
+  switch (operations) {
+    case '+':
+      return one.plus(two).toString();
+    case '-':
+      return one.minus(two).toString();
+    case '*':
+      return one.times(two).toString();
+    case '÷':
+      if (two.toString() === '0') {
+        return 0;
+      }
```

Comment:
You should return something else....maybe `NaN` or `INFINITY` ......it's definitely not ZERO..... will look into it on milestone 5

Reply:
Thank you @shloch